climada/engine/cost_benefit.py

```diff
@@ -134,28 +136,40 @@ class CostBenefit():
         'impact' (Impact): impact instance
     """
 
-    def __init__(self):
+    def __init__(self,
+                 present_year: Optional[int] = DEF_PRESENT_YEAR,
+                 future_year: Optional[int] = DEF_FUTURE_YEAR,
+                 tot_climate_risk: Optional[float] = 0.0,
+                 units:  Optional[str] = 'USD',
```

Comment:
Arguments with immutable values and defaults are not `Optional`:
```suggestion
                 present_year: int = DEF_PRESENT_YEAR,
                 future_year: int = DEF_FUTURE_YEAR,
                 tot_climate_risk: float = 0.0,
                 units: str = 'USD',
```